packages/components/src/Menu/Menu.tsx

```diff
@@ -297,3 +293,13 @@ function Action({
     </button>
   );
 }
+
+function MenuPortal({ children }: { readonly children: React.ReactElement }) {
+  const mounted = useIsMounted();
+
+  if (!mounted?.current) {
```

Comment:
I appreciate the SSR consideration, thanks!

Reply:
no problemo!

there are a few things in this PR that are attempting to access either the `document` or `window` and I've put in some checks to hopefully address them - that said, I didn't explicitly test this on a Remix setup